com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/HDBakedReflectionSystem.cs

```diff
@@ -326,9 +327,8 @@ public static bool BakeProbes(IList<HDProbe> bakedProbes)
             var cubeRT = HDRenderUtilities.CreateReflectionProbeRenderTarget(cubemapSize);
 
             // Render and write the result to disk
-            for (int i = 0; i < bakedProbes.Count; ++i)
+            foreach (var probe in bakedProbes)
```

Comment:
iirc foreach cause GC.Alloc no ?

Reply:
IIRC Antoine have remove all of them to be able to have 0 GC.Alloc

Reply:
Most of the foreach are fine but some of them allocate memory due to the kind of iterator there are using. 
Whether this allocates memory or not will depend on the type of `bakedProbes`

Reply:
They are reference type, no value type. So there is no boxing during the foreach loop